src/App.tsx

```diff
@@ -15,35 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  Name = 'name',
```

Comment:
We are sorting not by name, but by alphabet. Not very proper naming.

Reply:
Agreed. ( not critical)